tools/benchmark_block_hashes.py

```
@@ -0,0 +1,196 @@
+    local = block_hashes(source, args.payload_key, args.block_count)
+    fine_count = args.block_count * args.refine_factor
+    fine = {}
+    for record in source[:args.mismatches]:
+        parent = slice_point_id_hash(record["id"]) % args.block_count
+        if parent not in fine:
+            records = [p for p in source if slice_point_id_hash(p["id"]) % args.block_count == parent]
+            fine[parent] = block_hashes(records, args.payload_key, fine_count)
+
+    def reconcile(client):
+        remote = client.hashes(args.block_count)
+        for theirs, ours in zip(remote["blocks"], local["blocks"]):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)